<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mandelbrotmenge Qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>1121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Scaling:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="scalingEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Recursion Depth:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recursionEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>endRe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endReEdit">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="startReEdit">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>-2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>startRe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>startIm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="startImEdit">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endImEdit">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>endIm</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
